.github/workflows/sync.yml

```
@@ -0,0 +1,43 @@
+name: Sync
+on:
+  schedule: # every friday at 12:00 UTC
+    - cron: "0 12 * * 5"
+  workflow_dispatch:
+jobs:
+  sync-docs:
+    permissions:
+      contents: write
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: recursive
+      - uses: dtolnay/rust-toolchain@master
+        with:
+          toolchain: nightly
+      - uses: Swatinem/rust-cache@v2
+      - run: git checkout -b sync-docs
+      - name: Sync Docs
+        run: cargo xtask overview
+      - run: |
+          git config user.name ${{github.actor}}
+          git config user.email "${{github.actor_id}}+${{github.actor}}@users.noreply.github.com"
+      - run: git add .
+      - run: >
+          git commit -m "docs: sync docs"
```

maybe this should be more "wordy"

```suggestion
          git commit -m "docs: sync overview docs"
```
?

---

I wouldn't mind it.